core/src/main/scala/kafka/network/SocketServer.scala

```diff
@@ -734,9 +730,21 @@ private[kafka] class Acceptor(val endPoint: EndPoint,
         val endThrottleTimeMs = e.startThrottleTimeMs + e.throttleTimeMs
         throttledSockets += DelayedCloseSocket(socketChannel, endThrottleTimeMs)
         None
+      case e: IOException =>
+        info(s"Encountered IOException, closing connection", e)
```

Comment:
Should we log this as an error? Moreover, I would rephrase a bit the log: `Encountered an error while configuring the connection, closing it.`

Reply:
Right. I've pushed a commit.